docs/pages/branding/home.tsx

```diff
@@ -1,31 +1,48 @@
 import * as React from 'react';
-import { ThemeProvider } from '@material-ui/core/styles';
+import ThemeProvider from 'docs/src/modules/ThemeContext';
 import CssBaseline from '@material-ui/core/CssBaseline';
 import Typography from '@material-ui/core/Typography';
+import Divider from '@material-ui/core/Divider';
 import AppHeader from 'docs/src/layouts/AppHeader';
 import Container from '@material-ui/core/Container';
 import Hero from 'docs/src/components/home/Hero';
-import brandingTheme from 'docs/src/modules/brandingTheme';
 import References from 'docs/src/components/home/References';
+import ProductSuite from 'docs/src/components/home/ProductSuite';
+import ValueProposition from 'docs/src/components/home/ValueProposition';
+import DesignSystemComponents from 'docs/src/components/home/DesignSystemComponents';
+import Testimonials from 'docs/src/components/home/Testimonials';
+import Sponsors from 'docs/src/components/home/Sponsors';
+import HeroEnd from 'docs/src/components/home/HeroEnd';
+import AppFooter from 'docs/src/layouts/AppFooter';
 
 export default function Home() {
   return (
-    <ThemeProvider theme={brandingTheme}>
+    <ThemeProvider>
       <CssBaseline />
       <AppHeader />
       <Hero />
       <Container sx={{ py: 8 }}>
         <References />
         <Typography
-          variant="body2"
           color="grey.800"
+          maxWidth={450}
           textAlign="center"
-          sx={{ mt: 4, mx: 'auto', maxWidth: 360 }}
+          mx="auto"
+          mt={4}
+          variant="body2"
         >
-          From startups to Fortune 500s, the world&apos;s best teams leverage MUI to build their
-          UIs.
+          From startups to Fortune 500s, the world&apos;s best product teams leverage MUI to build
+          their UIs.
         </Typography>
       </Container>
+      <ProductSuite />
+      <ValueProposition />
+      <DesignSystemComponents />
+      <Testimonials />
+      <Sponsors />
+      <HeroEnd />
+      <Divider />
```

Comment:
Couldn't find where you declare the Divider's dark mode color, but just point this out for us to change it to `blue-dark-700` when in dark mode.

Reply:
fixed! you can search the file `brandingTheme.ts`. It includes all the design tokens and themed components